test/org/rust/lang/structure/RustStructureViewTest.kt

```diff
@@ -0,0 +1,28 @@
+package org.rust.lang.structure
+
+import com.intellij.openapi.util.io.FileUtil
+import com.intellij.testFramework.PlatformTestUtil.assertTreeEqual
+import org.rust.lang.RustTestCase
+import java.io.File
+
+class RustStructureViewTest : RustTestCase() {
+    override fun getTestDataPath() = "testData/structure"
+
+    private fun doTest(expected: String) {
+        myFixture.configureByFile(fileName);
+        myFixture.testStructureView { assertTreeEqual(it.tree, expected) }
+    }
+
+    private fun doFileTest() {
+        val text = FileUtil.loadFile(File(testDataPath + "/" + fileName.replace(".rs", ".txt")))
```

Comment:
I don't know, I don't have a Windows machine here to test this. I assumed that since `getTestDataPath()` was already using `/` too this would be okay.